### DISCUSSION TITLE
6E8719CA-3216-46C0-8478-CB7F81FC0B49

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#59] [General]
# Stephenson 2-18

Stephenson 2-18 is a red supergiant star located in the constellation Scutum, close to the open cluster Stephenson 2, located 19,000 light-years away from Earth. It is possibly the largest known star, with a radius of 2,150 solar radii. If it were placed at the center of the Solar System, its photosphere would encircle Saturn's orbit.

https://celestia.mobi/addon?item=6E8719CA-3216-46C0-8478-CB7F81FC0B49&lang=en

**Comment by orca81** [comment]
Kind of broken

**Comment by 13940951572** [comment]
你好？有没有盾牌座UY

**Comment by EricNelson28** [comment]
Good addon in quality, but however, it's not approved since the radius of this star (along with UY Scuti) was taken from an unreliable calculation.
It was based off of a bolometric luminosity of 437,000 x Sol's luminosity, as another method calculated it with a value of only 90,000 solar luminosity.
Either calculation is unreliable, therefore the size of the star as simulated in this addon is unconfirmed, so we have to wait until future technology gives us calculations of it (and other stars including UY Scuti) with reasonable certainty.

**Reply by EricNelson28** [reply]
Plus, there's a seam in the texture, which also adds to the lack of realism.

**Reply by EricNelson28** [reply]
By confirmed, I meant disputed.

**Reply by EricNelson28** [reply]
I meant unconfirmed.

**Comment by EricNelson28** [comment]
In addition, I mentioned the luminosity estimate of 437,000 x solar luminosity combined with the temperature estimate of 3,200K resulted in the radius estimate of 2,150 x solar radii.
It's great effort (and inspired me for a few addons - giving credit to you ), but since the methods used to calculate this star were unreliable (hence a disputed radius, temperature, and luminosity estimates), the addon is as well.

**Reply by Pascualito1008** [reply]
oh yeah She's a Big star

**Reply by Pascualito1008** [reply]
Less than UY scuti

**Reply by Pascualito1008** [reply]
And gets more bigger

**Reply by Pascualito1008** [reply]
The end.

**Comment by Pascualito1008** [comment]
EricNelson28 You cool

**Comment by Mathew1312** [comment]
Great
